flash/vision/classification/data.py

```diff
@@ -285,6 +256,7 @@ def default_train_transforms():
         if _KORNIA_AVAILABLE and not os.getenv("FLASH_TESTING", "0") == "1":
             #  Better approach as all transforms are applied on tensor directly
             return {
+                "to_tensor_transform": torchvision.transforms.ToTensor(),
```

Comment:
```suggestion
                "to_tensor_transform": lambda x: K.image_to_tensor(x, keepdim=True),
```
we could even make a module for that in case lambda is a problem

Reply:
Let's make those changes in another PR as I would have to change the weights.

Reply:
lambda is a problem since it cannot be pickled.